nix/postgres.nix

```diff
@@ -44,6 +44,37 @@ in
           default = "./data/${config.name}";
           description = "The DB data directory";
         };
+
+        pgHbaConf =
```

Comment:
I'd rename this to `hbaConf`, because the `pg` prefix is redundant (this is already inside `serivices.postgresql`).